src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/PreferStringContainsOverIndexOfTests.cs

```diff
@@ -728,12 +772,25 @@ private void TestMethod(string str, System.StringComparison comparison)
             }
         } 
     } 
+}";
+            string csFix = @"
+namespace TestNamespace 
+{ 
+    class TestClass 
+    { 
+        private void TestMethod(string str, System.StringComparison comparison) 
+        { 
+            if (" + notString + @"str.Contains(""This"", comparison)" + @")
```

Comment:
Can we add tests with named arguments with different combination/ordering of arguments in input:
1. `if ([|str.IndexOf(value: ""This"", comparison)" + operatorKind + value + @"|])`
2. `if ([|str.IndexOf(""This"", comparisonType: comparison)" + operatorKind + value + @"|])`
3. `if ([|str.IndexOf(value: ""This"", comparisonType: comparison)" + operatorKind + value + @"|])`
4. `if ([|str.IndexOf(comparisonType: comparison, value: ""This"")" + operatorKind + value + @"|])`

Reply:
Done. The only issue was for case 3 in VB. I've commented in the unit test